src/browser/web-view.js

```diff
@@ -237,7 +237,6 @@ define((require, exports, module) => {
     return IFrame.view({
       id: `web-view-${loader.id}`,
       uri: location,
-      className: `web-view ${isSelected ? 'selected' : ''}`,
```

Comment:
Nit: I found it useful with an devtools inspector to identify which one is selected.